src/planscape/forsys/views.py

```diff
@@ -79,6 +83,20 @@ def convert_dictionary_of_lists_to_rdf(
     return rdf
 
 
+def convert_rdfs_vector_to_dict_of_lists(raw_forsys_output: "rpy2.robjects.vectors.ListVector") -> dict:
```

Comment:
nit: can this be named to something along the lines of, "convert_r_listvector_to_python_dict"?

Being picky about semantics because the output is a dict of dicts of lists.